babel.config.json

```diff
@@ -1,3 +1,19 @@
 {
-	"extends": "./node_modules/eclipsefdn-solstice-assets/babel.config.json"
+	"extends": "./node_modules/eclipsefdn-solstice-assets/babel.config.json",
+	"overrides": [
```

Comment:
This should be put in the solstice assets babel config, as all sites would benefit from better polyfills

Reply:
Actually, this does not work 😕 It should be working, while the  polyfill is not imported at all. I tried directly adding <script> with corejs CDN, it works. This confused me a lot. I posted a comment in https://github.com/JeffreyWay/laravel-mix/issues/2592#issuecomment-768327759 .

Reply:
I pushed the code to keep a record of what I have tried.